### PR TITLE
Remove multiselect border on mobile

### PR DESCRIPTION
**Pull request by Primdahl26**
Makes mobile version look like this

Does not change desktop view

### CODE REVIEW

**Comment by AndreasPB** [comment]
Should something happen to the desktop version as well?

**Comment by Primdahl26** [comment]
> Should something happen to the desktop version as well?

Nope, desktop is the same.

**Comment by Primdahl26** [comment]
> > Should something happen to the desktop version as well?
> 
> Nope, desktop is the same.

But i think desktop should stay the same

File: frontend/src/routes/index.svelte [hunk 1]
```diff
@@ -160,7 +160,9 @@
 
              --borderRadius: var(--rounded-btn, .5rem);
              --background: {DT[`[data-theme=${$chosenTheme}]`]['base-100']};
-             --border: 1px solid {DT[`[data-theme=${$chosenTheme}]`]['primary']};
+             --border: 1px solid {DT[`[data-theme=${$chosenTheme}]`][
+      `${viewPortWidth <= 640 ? '' : 'primary'}`
```
Comment (AndreasPB) [inline frontend/src/routes/index.svelte:164]: Could we do this with the Navbar MediaQuery idea instead, so it happens dynamically?

Also, both the multi-selects should probably be a component - Would look so nice if index didn't have this crazy CSS madness

**Comment by Primdahl26** [comment]
We can reopen if we actually want this to happen

**Pull request closed without merging**